mmdet/datasets/pipelines/transforms.py

```diff
@@ -1903,3 +1905,195 @@ def __repr__(self):
                      else f'cutout_shape={self.candidates}, ')
         repr_str += f'fill_in={self.fill_in})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class RandomAffine:
+    """Random affine transform data augmentation.
+
+    This operation randomly generates affine transform matrix which including
+    rotation, translation, shear and scaling transforms.
+
+    Args:
+        rotate_degree (float): Maximum degrees of rotation transform.
```

Comment:
Can we add `max` prefix for `rotate_degree` and `translate_ratio`，`shear_degree` ？